apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -1285,3 +1286,108 @@ export const getIDPConnectedApps = (idpId: string): Promise<any> => {
             return Promise.reject(error);
         });
 };
+
+/**
+ * Get all sms notification senders with name SMSPublisher.
+ *
+ * @returns  A promise containing the response.
+ */
+export const getSMSNotificationSenders = (): Promise<any> => {
```

Comment:
Lets use SWR for caching.

Ref: https://github.com/wso2/identity-apps/blob/ab1e036b8d817fe2f6ad4fd1b1268e8b610af65c/apps/console/src/features/applications/api/application.ts#L202

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a